src/service/resources-impl.js

```diff
@@ -1779,6 +1783,27 @@ export class Resource {
       this.initialLayoutBox_ = box;
     }
     this.layoutBox_ = box;
+
+    // Calculate whether the element is currently is or in `position:fixed`.
+    let isFixed = false;
+    if (this.isDisplayed()) {
+      const win = this.resources_.win;
+      const body = win.document.body;
+      const viewport = this.resources_.viewport_;
+      for (let n = this.element; n && n != body; n = n.offsetParent) {
```

Comment:
So, there are some differences in interpretation in browsers for `offsetParent`. But none that play any role in this case.